### PR TITLE
fix(Center Settings): Fixes intents, contact type, categories, orgs, partial stewards new-backend#29,#26,#27

### PR DESCRIPTION
**Pull request by wski**

### CODE REVIEW

**Comment by blaino** [comment]
Unit tests still passing?  There's an intents test...

File: app/js/components/management/mall/Stewards.jsx [hunk 1]
```diff
@@ -14,10 +14,24 @@ var Stewards = React.createClass({
     getDefaultProps: function () {
         return {
             title: 'Steward',
-            url: API_URL + '/api/self/profile?role=ORG_STEWARD',
+            url: API_URL + '/api/profile?role=ORG_STEWARD',
             getDisplayName: function (selectedRecord) {
                 return selectedRecord.displayName;
             },
+            getUsername: (d) => {
```
Comment (blaino) [inline app/js/components/management/mall/Stewards.jsx]: Any chance you could use more descriptive variable names than d and r?

File: app/js/components/management/mall/Stewards.jsx [hunk 2]
```diff
@@ -26,16 +40,16 @@ var Stewards = React.createClass({
                     displayName: {
                         disabled: true
                     },
-                    stewardedOrganizations: {
+                    stweardOrgs: {
```
Comment (blaino) [inline app/js/components/management/mall/Stewards.jsx]: spelling. Are you sure this thing is working?

File: app/js/components/management/mall/Stewards.jsx [hunk 3]
```diff
@@ -26,16 +40,16 @@ var Stewards = React.createClass({
...
                     }
                 }
             },
             grid: {
                 columns: [
-                    { field: 'displayName', caption: 'Display Name', size: '34%' },
+                    { field: 'displayName', caption: 'Display Name', size: '33%' },
                     { field: 'username', caption: 'Username', size: '33%' },
-                    { field: 'stewardedOrganizations', caption: 'Steward Organizations', size: '33%'}
+                    { field: 'stweardOrgs', caption: 'Steward Organizations', size: '33%'}
```
Comment (blaino) [inline app/js/components/management/mall/Stewards.jsx]: spelling.

File: app/js/components/management/mall/Stewards.jsx (hunk 2 above)
Comment (wski) [inline app/js/components/management/mall/Stewards.jsx]: Yeah, because the other one is spelled wrong too

**Comment by wski** [comment]
Closing this PR for a later date, fixing this will require fixes to createEdit among other things, will just keep it open till everything else is fixed

**Pull request closed without merging**